ddtrace/contrib/trace_utils.py

```diff
@@ -158,7 +159,7 @@ def set_http_meta(
         span._set_str_tag(http.METHOD, method)
 
     if url is not None:
-        span._set_str_tag(http.URL, url)
+        span._set_str_tag(http.URL, strip_query_string(url))
```

Comment:
As per the discussion below, I think we should not strip the URL if `integration_config.trace_query_string` is `True`

Reply:
Could we also have tests for this if we choose to go ahead with it?

Reply:
So just to make sure, we do want the query string as part of the URL when `trace_query_string` is set, even though it would get traced a few lines below anyway with

~~~ python
    if query is not None and integration_config.trace_query_string:
        span._set_str_tag(http.QUERY_STRING, query)
~~~
Is this correct? If so, I'll make the change

Reply:
Correct, I think there's value in having the tag have the full URL (say if someone wanted to index their spans based off of the whole URL). Might be worth running by @brettlangdon too.

Reply:
let's do both, anyplace we are tagging `http.query_string` today, do not remove that tagging, but adding into `http.url` is 👍🏻